python/ray/train/huggingface/huggingface_predictor.py

```diff
@@ -74,7 +74,8 @@ def __init__(
                 "prediction will only use CPU. Please consider explicitly "
                 "setting `HuggingFacePredictor(use_gpu=True)` or "
                 "`batch_predictor.predict(ds, num_gpus_per_worker=1)` to "
-                "enable GPU prediction."
+                "enable GPU prediction. Ignore if you have set `device` or "
+                "`device_map` arguments in the `pipeline` manually."
```

Comment:
Would it be possible to check for this and not log a warning?
> if you have set `device` or `device_map` arguments in the `pipeline` manually

Reply:
it is possible for `device`, but `device_map` would require usage of private APIs

Reply:
Got it. Makes sense